purldb-toolkit/README.rst

```diff
@@ -20,84 +25,541 @@ Use this command to get basic help::
 
     $ purlcli --help
     Usage: purlcli [OPTIONS] COMMAND [ARGS]...
-    
+
       Return information from a PURL.
-    
+
     Options:
       --help  Show this message and exit.
-    
+
     Commands:
       metadata  Given one or more PURLs, for each PURL, return a mapping of...
       urls      Given one or more PURLs, for each PURL, return a list of all...
-      validate  Check the syntax of one or more PURLs.
+      validate  Check the syntax and upstream repo status of one or more PURLs.
       versions  Given one or more PURLs, return a list of all known versions...
 
 
 And the following subcommands:
 
-- Validate a PURL::
+'validate' -- validate a PURL
```

Comment:
```suggestion
``validate`` : validate a PURL
```
Looks better this way. It's better to wrap command names in double ` to highlight

Reply:
Consider the same for all other headings

Reply:
Nice improvement -- done.